### ISSUE TITLE
Is there any problem if I use latest version eslint-plugin-vue?

### ISSUE THREAD

**Issue by ZMDreamer** [vuejs/eslint-plugin-vue#748]
**Tell us about your environment**	
* **ESLint version:**	
* **eslint-plugin-vue version:**	
* **Node version:**	

**Please show your full configuration:**

```json

```

**What did you do?**	

```vue

```

**What did you expect to happen?**

**What actually happened?**

**Comment by michalsnik** [comment]
Please reopen once you provide more informations.

**Comment by ZMDreamer** [comment]
![image](https://user-images.githubusercontent.com/39700194/50744466-57285e80-125e-11e9-9d41-d0449749f592.png)
Do you see the errors? When I use ‘Ctrl + S‘ key the  code will format itself by the rules that I define. But now it work in the wrong way. It worked well before I updated the node_modules file. So I want to know whether it has problem in the latest eslint-plugin-vue package or I should add some other config in the '.eslintrc.js' file.

**Comment by ZMDreamer** [comment]
Thank you for your reply.

**Comment by ota-meshi** [comment]
@ZMDreamer Hello!
In the image, I think like the error result of [eslint-plugin-prettier](https://github.com/prettier/eslint-plugin-prettier).
The error code `prettier/prettier` is the error code of `eslint-plugin-prettier`.
The error code by `eslint-plugin-vue` starts with `vue/`.

**Comment by ZMDreamer** [comment]
Thank you for your repaly. I will look at the problems in the way you tell me .
